guides/plugins/plugins/administration/using-vuex-state.md

```diff
@@ -0,0 +1,113 @@
+# Using Vuex Stores
+
+The Shopware 6 Administration uses [Vuex](https://vuex.vuejs.org/) stores to keep track of complex state, while just adding a wrapper around it.
+Learn what Vuex is, how to use it and when to use it from their great [documentation](https://vuex.vuejs.org/).
+This guide will show you how to use Vuex as you normally would, through the interfaces provided by the Shopware 6 Administration.
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance, the files and preferably a registered module. 
+Of course you'll have to understand JavaScript and have a basic familiarity with [Vue](https://vuejs.org/) the framework used in the Administration and it's flux library [Vuex](https://vuex.vuejs.org/).
+
+## Creating a store
+
+The creation of a store is the same as in normal Vuex with the only limitation that all stores have to be `namespaced` in order to prevent collisions with other third party plugins or the Shopware 6 Administration itself.
+
+The following code snippet is the `namespaced` store we will register later through Shopware to the underlying Vuex.
+It is admittedly rather short and has only one variable called `content` and a setter for it, but again this all the same as in Vuex.
```

Comment:
If I unterstood correctly, the `namespaced: true` part is absolutely necessary ? 
Than either write it here again or put a comment in the code example

```suggestion
It is admittedly rather short and has only one variable called `content` and a setter for it, but again this all the same as in Vuex. Beware of the property `namespaced`, though.
```

Reply:
i'll consult @jleifeld and add a sentence about it